modules/migration-viewing-migration-crs.adoc

```diff
@@ -1,20 +1,64 @@
 // Module included in the following assemblies:
 //
-// migration/migrating-3-4/migrating-openshift-3-to-4.adoc
-// migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc
-// migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc
+// migration/migrating-3-4/troubleshooting.adoc
+// migration/migrating-4_1-4/troubleshooting.adoc
+// migration/migrating-4_2-4/troubleshooting.adoc
 [id='migration-viewing-migration-crs_{context}']
-= Viewing migration custom resources
+= Viewing migration Custom Resources
 
-To view a migration custom resource:
+The CAM tool creates the following Custom Resources (CRs) for migration:
```

Comment:
It looks like CRs was already spelled out in the intro, so I think it should just be `CRs` here.